nltk/translate/api.py

```diff
@@ -163,51 +157,6 @@ def invert(self):
         return AlignedSent(self._mots, self._words,
                                self._alignment.invert())
 
-    def precision(self, reference):
-        """
-        Return the precision of an aligned sentence with respect to a
-        "gold standard" reference ``AlignedSent``.
-
-        :type reference: AlignedSent or Alignment
-        :param reference: A "gold standard" reference aligned sentence.
-        :rtype: float or None
-        """
-        # Get alignments in set of 2-tuples form
-        # The "possible" precision is used since it doesn't penalize for finding
-        # an alignment that was marked as "possible" (NAACL corpus)
-
-        align = self.alignment
-        if isinstance(reference, AlignedSent):
-            possible = reference.alignment
-        else:
-            possible = Alignment(reference)
-
-        return precision(possible, align)
-
-
-    def recall(self, reference):
-        """
-        Return the recall of an aligned sentence with respect to a
-        "gold standard" reference ``AlignedSent``.
-
-        :type reference: AlignedSent or Alignment
-        :param reference: A "gold standard" reference aligned sentence.
-        :rtype: float or None
-        """
-        # Get alignments in set of 2-tuples form
-        # The "sure" recall is used so we don't penalize for missing an
-        # alignment that was only marked as "possible".
-
-        align = self.alignment
-        if isinstance(reference, AlignedSent):
-            sure = reference.alignment
-        else:
-            sure  = Alignment(reference)
-
-        # Call NLTKs existing functions for recall
-        return recall(sure, align)
-
-
     def alignment_error_rate(self, reference, possible=None):
```

Comment:
Just a stray observation. Why is there a default argument for `possible`? Probably should be removed